# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../atop/install.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- atop-1.21/Makefile.vanilla	2007-10-06 12:33:14.000000000 +0200
+++ atop-1.21/Makefile	2007-10-06 12:33:43.000000000 +0200
@@ -39,5 +39,5 @@
 		if [ ! -d $(DESTDIR)$(CRNPATH) ]; then mkdir -p $(DESTDIR)$(CRNPATH); fi
 		if [ ! -d $(DESTDIR)$(ROTPATH) ]; then mkdir -p $(DESTDIR)$(ROTPATH); fi
-		cp atop   	 $(DESTDIR)$(BINPATH)/atop
+		install atop   	 $(DESTDIR)$(BINPATH)/atop
 		chown root	 $(DESTDIR)$(BINPATH)/atop
 		chmod 04711 	 $(DESTDIR)$(BINPATH)/atop
@@ -50,5 +50,4 @@
 		touch            $(DESTDIR)$(LOGPATH)/dummy_before
 		touch            $(DESTDIR)$(LOGPATH)/dummy_after
-		if [ -z "$(DESTDIR)" ]; then /sbin/chkconfig --add atop; fi
 
 distr:
